plugins/modules/network_compliance_workflow_manager.py

```diff
@@ -0,0 +1,1340 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024, Cisco Systems
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""Ansible module to perform Network Compliance Operations on devices in Cisco Catalyst Center."""
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+__author__ = ("Rugvedi Kapse, Madhan Sankaranarayanan")
+
+DOCUMENTATION = r"""
+---
+module: network_compliance_workflow_manager
+short_description: Network Compliance module for managing network compliance tasks on reachable device(s) in Cisco Catalyst Center.
+description:
+- Perform compliance checks or sync configurations on reachable devices using IP Address(s) or Site.
+- API to perform full compliance checks or specific category checks on reachable device(s).
+- API to sync device configuration on device(s).
+version_added: '6.6.0'
+extends_documentation_fragment:
+  - cisco.dnac.workflow_manager_params
+author: Rugvedi Kapse (@rukapse)
+        Madhan Sankaranarayanan (@madhansansel)
+options:
+  config_verify:
+    description: Set to True to verify the Cisco Catalyst Center config after applying the playbook config.
+    type: bool
+    default: False
+  state:
+    description: State of Cisco Catalyst Center after module completion.
+    type: str
+    choices: [ merged ]
+    default: merged
+  config:
+    description: List of device details for running a compliance check or synchronizing device configuration.
+    type: list
+    elements: dict
+    required: True
+    suboptions:
+      ip_address_list:
+        description: List of IP addresses of devices to run a compliance check on or synchronize device configurations.
+                     Either 'ip_address_list' or 'site_name' is required for module to execute.
+                     If both 'ip_address_list' and 'site_name' are provided, 'ip_address_list' takes precedence.
```

Comment:
Can't we run the compliance for both? Devices from site_name and ip_address_list?

Reply:
Addressed!